src/Language/PureScript/Ide/Util.hs

```diff
@@ -90,14 +90,14 @@ encodeT = TL.toStrict . TLE.decodeUtf8 . encode
 decodeT :: (FromJSON a) => Text -> Maybe a
 decodeT = decode . TLE.encodeUtf8 . TL.fromStrict
 
-properNameT :: Iso' (P.ProperName a) Text
-properNameT = iso P.runProperName P.ProperName
+properNameT :: Getting r (P.ProperName a) Text
```

Comment:
We ended up not needing the full power of an iso here. Is that right?

Reply:
If all we need is a `SimpleGetter` then we might as well just use `to P.runProperName` in the usage.